stuffer/s2n_stuffer.h

```diff
@@ -164,6 +164,7 @@ int s2n_stuffer_copy(struct s2n_stuffer *from, struct s2n_stuffer *to, uint32_t
  * "FF" == 255 or [ 0xff ]
  * "0001" == 1 or [ 0x00, 0x01 ]
  */
+S2N_RESULT s2n_hex_digit(uint8_t nibble, uint8_t *hex_digit);
```

Comment:
Adding "stuffer" to the name to match the module makes this confusing, because it doesn't actually take a stuffer. But I'm not sure where else this function should live :/